sql/2022/webassembly/language_usage.sql

```diff
@@ -0,0 +1,27 @@
+# standardSQL
+# The counts of language usage across wasm modules.
+
+WITH wasm AS (
+  SELECT
+    *,
+    JSON_QUERY(payload, '$._wasm_stats') AS wasm_stats
+  FROM
+    `httparchive.almanac.requests`
+  WHERE
+    date = '2022-06-01' AND
+    (mimeType = 'application/wasm' OR ext = 'wasm')
+)
+
+SELECT
+  client,
+  JSON_VALUE(wasm_stats, '$.language') AS language,
+  COUNT(0) AS count
```

Comment:
I'd also suggest calculating this as a %
```suggestion
  COUNT(0) AS count,
  SUM(COUNT(0)) OVER (PARTITION BY client) AS total,
  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY client) AS pct
```